docs/antctl.md

```diff
@@ -181,12 +181,26 @@ antrea-network-policy.md#antrea-native-policy-ordering-based-on-priorities).
 antctl get networkpolicy --sort-by=effectivePriority
 ```
 
-Antrea Agent additionally supports printing NetworkPolicies applied to a
-specified local Pod using this `antctl` command:
+Antrea Agent additionally supports some functions using this `antctl` command.
 
-```bash
-antctl get networkpolicy -p pod -n namespace
-```
+* Printing NetworkPolicies applied to a specified local Pod.
+
+  ```bash
+  antctl get networkpolicy -p pod -n namespace
+  ```
+
+* Printing NetworkPolicies with a specific source type or source name. The source of a control plane NetworkPolicy is the original policy resource (K8s NetworkPolicy or Antrea-native Policy) from which the control plane NetworkPolicy was derived.
+  * With specific source type: K8sNP, ACNP, ANP.
+
+  ```bash
+  antctl get networkpolicy -T sourceType
+  ```
+
+  * With specific source name.
+
+  ```bash
+  antctl get networkpolicy -S sourceName
```

Comment:
This is the uid of the user-facing API object? If so, better clarify here

Reply:
This is the name of source NP and might use with `-n namespace`. I will clarify here.